mmcv/ops/csrc/common/cuda/diff_iou_rotated_cuda_kernel.cuh

```diff
@@ -0,0 +1,155 @@
+// Copyright (c) OpenMMLab. All rights reserved
+// Adapted from https://github.com/lilanxiao/Rotated_IoU/cuda_op/sort_vert_kernel.cu  # noqa
+#include <ATen/ATen.h>
```

Comment:
you can include 
```

#ifdef MMCV_USE_PARROTS
#include "parrots_cuda_helper.hpp"
#else
#include "pytorch_cuda_helper.hpp"
#endif
```

Instead of most other heads here. pytorch_cuda_helper would provide most tools you need.